packages/backend/src/graphql/schema.graphql

```diff
@@ -563,11 +562,6 @@ type License {
   verified: Boolean
 }
 
-type GetBillingAndUsage {
-  subscription: Subscription
-  usage: Usage
-}
-
 type MonthlyQuota {
```

Comment:
We should remove other types that are only used with `Subscription` as well, like MonthlyQuota, NextBIllAmount etc.